src/main/java/org/galatea/starter/domain/HistoricalPricesDB.java

```diff
@@ -0,0 +1,210 @@
+package org.galatea.starter.domain;
+
+import java.math.BigDecimal;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import javax.persistence.Table;
+
+@Entity
+@Table
+@IdClass(HistoricalPricesId.class)
+public class HistoricalPricesDB {
+
+  @Id
+  @Column
+  private String symbol;
+
+  @Id
+  @Column
+  private String date;
+
+  @Id
+  @Column
+  private String minute;
+
+  private BigDecimal close;
+  private BigDecimal high;
+  private BigDecimal low;
+  private BigDecimal open;
+  private BigDecimal volume;
+
+  /**
+   * Default Constructor.
+   */
+  public HistoricalPricesDB() {
+    //Does Nothing
+  }
+
+  /**
+   * Constructor that initializes all field variables.
+   *
+   * @param symbol field symbol
+   * @param date field date
+   * @param close field close
+   * @param high field high
+   * @param low field low
+   * @param open field open
+   * @param volume field volume
+   * @param minute field minute
+   */
+  public HistoricalPricesDB(final String symbol, final String date, final BigDecimal close,
+      final BigDecimal high, final BigDecimal low, final BigDecimal open, final BigDecimal volume,
+      final String minute) {
+    this.symbol = symbol;
+    this.date = date;
+    this.close = close;
+    this.high = high;
+    this.low = low;
+    this.open = open;
+    this.volume = volume;
+    this.minute = minute;
+  }
+
+  /**
+   * Getter for symbol.
+   *
+   * @return symbol
```

Comment:
Don't manually create getters and setters. Look into lombok @Getter and @Setter, and @Data